src/main/java/pl/futurecollars/invoice/controller/InvoiceController.java

```diff
@@ -3,42 +3,38 @@
 import io.swagger.annotations.Api;
 import java.util.List;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.DeleteMapping;
-import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 import pl.futurecollars.invoice.model.Invoice;
 import pl.futurecollars.invoice.service.InvoiceService;
 
 @Api(tags = {"invoice-controller"})
```

Comment:
To również mogło by się znaleźć w InvoiceApi

Reply:
Przeniesione

Reply:
W takim razie jeszcze nie wypuszowane :>

Reply:
Rozwiązując konflikty narobiłem bałaganu, ale już wrócił porządek i jest już wypuszowana zmiana.